providers/wallet.csv

```diff
@@ -46,6 +46,16 @@ c98wallet,C98wallet,FALSE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,FA
 wallet3,Wallet 3,TRUE,"[""iOS"",""Android""]",FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,TRUE,FALSE,"[""ERC-4337""]",FALSE,$0,"[""discord""]","[""audit by third parties""]","[""EN""]",FALSE,"[""[Website](https://wallet3.io/)""]"
+cwallet,Cwallet,FALSE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,"[""ERC-20"",""BEP-2"",""BEP-20"",""ERC-721"",""SPL"",""TRC-20"",""TRC-10"",""BRC-20""]",TRUE,$0 ,"[""email"",""24/7 live chat"",""x"",""discord"",""telegram"", ""webform""]","[""audit by third parties""]","[""EN"", ""24 Others""]",TRUE,"[""[Website](https://cwallet.com/)""]"
+ascoin-wallet,AScoin Wallet,FALSE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,TRUE,TRUE,FALSE,TRUE,TRUE,TRUE,FALSE,"[""ERC-20"",""BEP-2"",""BEP-20"",""ERC-721"",""SPL"",""TRC-20"",""TRC-10"",""BRC-20""]",TRUE,Paid,"[""email"",""x"",""discord"",""telegram""]","[""audit by third parties""]","[""EN"",""RU"",""JA"",""KO"",""VI"",""PT"",""AR"",""ZH"",""FR"",""ES"",""KO"",""ID"",""UK"",""TI""]",FALSE,"[""[Website](https://www.ascoin.pro/)""]"
+blocto-wallet,Blocto Wallet,TRUE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,TRUE,TRUE,FALSE,TRUE,TRUE,TRUE,FALSE,"[""4337""]",TRUE,Paid,"[""email"",""x"",""discord"",""telegram"", ""webform""]","[""audit by third parties""]","[""EN"",""RU"",""JA"",""KO"",""PT"",""ZH"",""ES"",""KO""]",TRUE,"[""[Website](https://blocto.io/)""]"
+ownbit,Ownbit,FALSE,"[""iOS"",""Android""]",FALSE,TRUE,TRUE,FALSE,TRUE,FALSE,TRUE,TRUE,"[""ERC-20"",""BEP-2"",""BEP-20"",""ERC-721"",""SPL"",""TRC-20"",""TRC-10"",""BRC-20""]",TRUE,$0 ,"[""email"",""x"",""telegram""]","[""audit by third parties""]","[""EN"",""ES"",""NL"",""FL""]",TRUE,"[""[Website](https://ownbit.io/en/)""]"
+cake-wallet,Cake Wallet,FALSE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,TRUE,FALSE,TRUE,TRUE,TRUE,TRUE,FALSE,"[""ERC-20"",""BEP-2"",""BEP-20"",""ERC-721"",""SPL"",""TRC-20"",""TRC-10"",""BRC-20""]",FALSE,$0 ,"[""email"",""telegram""]","[""audit by third parties""]","[""EN"",""FR"",""ES"",""DE"",""ZH"",""NL"",""IT"",""PL"",""PT"",""JA"",""IT"",""ID"",""HI"",""NL"",""CS"",""CA"",""BG"",""AR"",""VI"",""TR"",""RU"",""FA"",""KO"",""SV"", ""HA"", ""YO"", ""UR"", ""HR"", ""MY""]",TRUE,"[""[Website](https://cakewallet.com/)""]"
+fluvi-wallet,Fluvi Wallet,FALSE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,TRUE,FALSE,TRUE,FALSE,FALSE,TRUE,FALSE,"[""ERC-20"",""BEP-2"",""BEP-20"",""ERC-721"",""SPL"",""TRC-20"",""TRC-10"",""BRC-20""]",TRUE,$0 ,"[""email""]","[""audit by third parties""]","[""EN""]",FALSE,"[""[Website](https://fluviwallet.xyz/)""]"
+wigwam-wallet,Wigwam Wallet,TRUE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,TRUE,FALSE,TRUE,TRUE,FALSE,TRUE,FALSE,"[""ERC-20"",""BEP-2"",""BEP-20"",""ERC-721"",""SPL"",""TRC-20"",""BRC-20""]",FALSE,$0 ,"[""email"",""x""]","[""audit by third parties""]","[""EN"",""ES"",""FR"",""IT"", ""RU"",""UK"",""ZH"",""JA"",""KO""]",TRUE,"[""[Website](https://wigwam.app/)""]"
```

Comment:
Starred column should always be false as per style guide (https://github.com/Chain-Love/chain-love/wiki/Common-Columns#starred)